REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,292 @@
+them to be restarted if they are killed by the operating system.  To use replication policies, you need to subclass
+one of the existing replication Service components and implement the specifics of your required replications.
+
+##Creating your service
+There are two options when creating your service. 
+
+1. If you don't require periodic replications, you should create a subclass of `ReplicationService`.
+2. If you want your replications to repeat at (roughly) fixed time intervals, as well as possibly adding other criteria that
+are required for replication, you should create a subclass of `PeriodicReplicationService`. This class takes care of the
+scheduling of periodic alarms to trigger the replications. This also requires you to implement a `BroadcastReceiver` that is a subclass of `PeriodicReplicationReceiver`. 
+
+Both Services allow other application components to bind to them. This allows components to be notified when
+replication is complete and also allows the rate of any periodic replication required by the application to be varied
+depending on whether there are components bound to the service or not. For example, it may be desirable to have an app that
+triggers replications every 24 hours in the background when the app is not actively being used, but when the app is being
+used and a particular `Activity` is displayed it may be desirable to replicate every few minutes and then update the
+`Activity` to show the data when replication has completed.
+
+###ReplicationService
+`ReplicationService` is an abstract class and your subclass must implement the following abstract method:
+* `protected abstract Replicator[] getReplicators(Context context);` This method should return an array of `Replicator` objects that define the replications you want to invoke.
+
+###PeriodicReplicationService
+This abstract class is a child of `ReplicationService` and requires you to implement the following abstract methods:
+* `protected abstract Replicator[] getReplicators(Context context);` This method should return an array of `Replicator` objects that define the replications you want to invoke.
+* `protected abstract int getBoundIntervalInSeconds();` This method should return the interval (in seconds) you wish to have between replications when components are bound to the Service.
+* `protected abstract int getUnboundIntervalInSeconds();` This method should return the interval (in seconds) you wish to have between replications when components are not bound to the Service.
+* `protected abstract boolean startReplicationOnBind();` This should return `true` if you wish to have replications triggered immediately when a component binds to the Service.
+
+Note that internally this uses `AlarmManager.setInexactRepeating()` to schedule the repetition of the replications at the given intervals in a battery efficient way, this means the intervals between replications will not be exact.
```

Comment:
What did you think about using a fully qualified name for this? Maybe a link to the Android doc so people can find the information easily about how inexact this is.